packages/forms-web-app/__tests__/unit/routes/submit-appeal/index.test.js

```diff
@@ -16,6 +17,7 @@ describe('routes/submit-appeal', () => {
 
   it('should define the expected routes', () => {
     expect(use).toHaveBeenCalledWith(enterAppealDetailsRouter);
-    expect(use.mock.calls.length).toBe(1);
+    expect(use).toHaveBeenCalledWith(applicationSavedRouter);
+    expect(use.mock.calls.length).toBe(2);
```

Comment:
This assertion is not very useful imo

Reply:
happy to remove - I think I can understand a reasoning behind adding the count as a reminder to update this test, but if using TDD there is no reason to include it